lab-padget/test/server-test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// npm run test
+console.log('Evaluating server.js');
+
+const expect = require('chai').expect;
+console.log(expect);
+// const assert = require('assert');
+// const reqContext = require('../lib/server.js');
+
```

Comment:
if you create an instance of your server in a before block, it'll be available to each it block. in those it blocks, you can, and should, test each command (for full credit)